tests/integration/gha-run.sh

```diff
@@ -93,6 +93,9 @@ function run_tests() {
     kubectl -n kube-system wait --timeout=10m --for=condition=Ready -l name=kata-deploy pod
     kubectl apply -f "${tools_dir}/packaging/kata-deploy/runtimeclasses/kata-runtimeClasses.yaml"
 
+    echo "\n\nGather information about the nodes and pods after having kata-deploy ready"
```

Comment:
I'm curious about those '\n\n' in the begin. Does Github parse it differently? or you are missing the `-e` to echo?